app/views/users/confirmations/invite_admin.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :page_title, "Invite a team member to be an administrator" %>
+
+<%= form_with model: resource, as: resource_name, url: invitation_path(resource_name) do |f| %>
+  <% if @errors %>
+    <%= f.govuk_error_summary %>
+  <% end %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-l">Invite a team member to be an administrator</h1>
+      <div class="govuk-inset-text">
+        There must be a minimum of 2 administrators to add IPs<br /> or multiple locations.
+      </div>
+      <p class="govuk-body">
+        You can skip this step for now, but you’ll need to add another admin before you can access the full functionality of GovWifi.
+        <br />
+        <br />
+        Two admins are required so that you can access your account if you lose your 2FA details or device. It also helps ensure that there’s a contact for GovWifi in your organisation if someone leaves.
+        <p class="govuk-body">Administrators can:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>view locations and IPs, team members, and logs</li>
+          <li>manage locations and IPs</li>
+          <li>add or remove team members</li>
+        </ul>
+      </p>
+
+      <h1 class="govuk-heading-l">Invite a team member</h1>
+      <div class="govuk-grid-column-full govuk-!-padding-left-0">
+        <%= f.fields_for :user do |user_fields| %>
+          <%= user_fields.govuk_email_field :email, label: { text: "Email address of team member" } %>
+        <% end %>
+        <%= hidden_field_tag :permission_level, "administrator" %>
```

Comment:
Maybe better use the form model for the hidden field tags; ` <%= f.hidden_field :permission_level, value: "administrator" %>`